src/iframe-helper.js

```diff
@@ -311,6 +311,20 @@ export function listenForOnce(iframe, typeOfMessage, callback, opt_is3P) {
 }
 
 /**
+ * Return a promise that resolve when listenForOnce function has listened to the message
```

Comment:
`Returns a promise that resolves when the given message has been observed for the first time.`